homeassistant/components/systemmonitor/sensor.py

```diff
@@ -77,6 +99,7 @@
         DATA_RATE_MEGABYTES_PER_SECOND,
         "mdi:server-network",
         True,
```

Comment:
Device class is True?

Reply:
Will check it when I get home from Easter

Reply:
It was off by one and I append the `True` to the end without realizing it.  Good catch.  All fixed